docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW TABLET STORAGE FORMAT.md

```diff
@@ -0,0 +1,53 @@
+---
+{
+    "title": "SHOW TABLET STORAGE FORMAT",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# SHOW TABLET STORAGE FORMAT
+## description
+    该语句用于显示Backend上的存储格式信息（仅管理员使用）
+    语法：
+        SHOW TABLET STORAGE FORMAT [VERBOSE]
+
+## example
+    MySQL [(none)]> show tablet storage format;
+    +-----------+---------+---------+
+    | BackendId | V1Count | V2Count |
+    +-----------+---------+---------+
+    | 10002     | 0       | 2867    |
+    +-----------+---------+---------+
+    1 row in set (0.003 sec)
+    MySQL [test_query_qa]> show tablet storage format verbose;
```

Comment:
show tablet storage format limit 10? 
If user find tablet id, and could use show tablet to find related table? And how to deal with the table? Any SQL to deal with the table?